lib/crypto.js

```diff
@@ -252,6 +252,10 @@ function getFipsForced() {
   return 1;
 }
 
+function getRandomValues(array) {
+  return randomFillSync(array);
```

Comment:
I think we should still use `require('internal/crypto/random').getRandomValues`. `randomFillSync` has a different array length limitation.

Reply:
Do you have an example?

Reply:
An example of the difference? I have not confirmed it with code but checked the documentation:

https://nodejs.org/dist/latest-v17.x/docs/api/webcrypto.html#cryptogetrandomvaluestypedarray

> An error will be thrown if the given typedArray is larger than 65,536 bytes.

https://nodejs.org/dist/latest-v17.x/docs/api/crypto.html#cryptorandomfillsyncbuffer-offset-size

> The size of the provided buffer must not be larger than 2**31 - 1.

Reply:
Oh here it is:

> If the byteLength of array is greater than 65536, throw a QuotaExceededError and terminate the algorithm.

Reply:
But again, since this alias is not web-compatible anyway, shouldn't we enforce this limit as well?

Reply:
Again, I think that the `this` limitation is not something that users who grab the alias would be affected by/think about. If that's the only difference, I think the alias makes sense. Otherwise, I agree with you and we should deprecate it in v17 and remove it in v18. But to do that we need to understand why @jasnell added it and what is his plan for the future of require('crypto')/globalThis.crypto

Reply:
FWIW `getRandomValues` implementation also calls `randomFillSync` internally:
https://github.com/nodejs/node/blob/b8de7aa4c2731883121bd7dcd19ec116f685eaa0/lib/internal/crypto/random.js#L321-L327



> Again, I think that the `this` limitation is not something that users who grab the alias would be affected by/think about. If that's the only difference, I think the alias makes sense. Otherwise, I agree with you and we should deprecate it in v17 and remove it in v18. But to do that we need to understand why @jasnell added it and what is his plan for the future of require('crypto')/globalThis.crypto

Alright, that makes sense, I'll update this PR and let James choose which PR should land.